.env

```diff
@@ -1,22 +0,0 @@
-## Local EVM Node Development
```

Comment:
@knoll3 -- can an `.env.example` file be created so there's a reference point for what goes in this file?